mr/src/test/java/org/apache/iceberg/mr/mapreduce/TestIcebergInputFormat.java

```diff
@@ -495,19 +526,22 @@ private DataFile writeFile(
             .named(fileFormat.name())
             .build();
         break;
-      case PARQUET:
-        appender = Parquet.write(Files.localOutput(file))
-            .schema(table.schema())
-            .createWriterFunc(GenericParquetWriter::buildWriter)
-            .named(fileFormat.name())
-            .build();
-        break;
+
```

Comment:
Usually these non-functional changes are avoided in Iceberg to reduce noise in PRs.